dpAutoRigSystem/dpAutoRig.py

```diff
@@ -1664,6 +1664,8 @@ def createBaseRigNode(self):
             cmds.setAttr(self.masterGrp+".name", self.masterGrp, type="string")
             # add date data log:
             cmds.addAttr(self.masterGrp, longName="lastModification", dataType="string")
+            # add model version:
+            cmds.addAttr(self.masterGrp, longName="modelVersion", attributeType="long")
```

Comment:
Add defaultValue and minValue = 0, please.

Reply:
Reviewed, thanks.